lib/Ed25519VerificationKey2020.js

```diff
@@ -154,15 +199,7 @@ export class Ed25519VerificationKey2020 extends LDKeyPair {
    * @returns {string} The fingerprint.
    */
   fingerprint() {
-    // ed25519 cryptonyms are multicodec encoded values, specifically:
-    // (multicodec ed25519-pub 0xed01 + key bytes)
-    const pubkeyBytes = this._publicKeyBuffer;
-    const buffer = new Uint8Array(2 + pubkeyBytes.length);
-    buffer[0] = 0xed;
-    buffer[1] = 0x01;
-    buffer.set(pubkeyBytes, 2);
-    // prefix with `z` to indicate multi-base base58btc encoding
-    return `z${base58btc.encode(buffer)}`;
+    return this.publicKeyMultibase;
```

Comment:
My only hesitancy here is that this function used to return a copy of the buffer view and this will return it directly. But, fewer copies FTW!

Reply:
If someone changes the value of the returned fingerprint... they deserve everything coming to them!